pkg/sitewise/datasource.go

```diff
@@ -123,7 +148,7 @@ func (ds *Datasource) invoke(ctx context.Context, _ *backend.QueryDataRequest, b
 
 func (ds *Datasource) HealthCheck(ctx context.Context, req *backend.CheckHealthRequest) error {
 
-	sw, err := ds.getClient(ctx, "") // Default region
+	sw, err := ds.getClient(ctx, ds.cfg.Region)
```

Comment:
Needed this to fix the "Save and Test" button in the config editor. The client needed to be configured with the Edge region instead of an empty region, otherwise I was getting a 403 error.